### PR TITLE
Add semantic categories to typography styles.

### PR DESCRIPTION
**Pull request by tommyshellberg**
#### Changes proposed in this Pull Request:
Add semantic descriptions of typography styles, similar to color styles, which allows for matching of meaning to typography("serif", "sans-serif", "modern", "clean", etc.)

### CODE REVIEW

File: assembler/styles/typography/01-messapia-bold-bricolage.json [hunk 1]
```diff
@@ -1,7 +1,15 @@
 {
 	"$schema": "https://schemas.wp.org/trunk/theme.json",
-	"version": 2,
+	"version": 3,
```
Comment (mdawaffe) [inline assembler/styles/typography/01-messapia-bold-bricolage.json:3]: I ran the file through a validator with the version 3 schema, and the only error was the `categories` (as expected).

**Review by mdawaffe — APPROVED** [review]
LGTM

**Comment by tommyshellberg** [comment]
@jeherve would you perhaps be able to help with merging + deploying this?
I'm not quite sure how version bumping is done, does a script bump theme versions or do I need to update versions manually? 🤔

**Comment by jeherve** [comment]
@tommyshellberg I'm not too familiar with how deployments work from this repo, but it looks like this is handled daily by the Themes team:
https://github.com/Automattic/themes/blob/trunk/CONTRIBUTING.md#deployments

I think you could merge the PR, but you can always reach out in #themes in Slack to get some clarification on that.

**Pull request merged**